setup.py

```diff
@@ -20,19 +20,19 @@
 
 
```

Comment:
This commit is taken from https://github.com/huggingface/optimum-habana/pull/1979